azurerm/internal/services/datafactory/data_factory_dataset_json_resource.go

```diff
@@ -92,7 +92,7 @@ func resourceArmDataFactoryDatasetJSON() *schema.Resource {
 				Type:     schema.TypeList,
 				MaxItems: 1,
 				Optional: true,
-				//ConflictsWith: []string{"sftp_server_location", "file_server_location", "s3_location", "blob_storage_location"},
+				//ConflictsWith: []string{"sftp_server_location", "file_server_location", "s3_location", "azure_blob_storage_location"},
```

Comment:
The document of this resource lacks of this `azure_blob_storage_location` field, would you please also complement the document so as user are aware of this feature?

Reply:
You're right. I added to Delimited text but not the json dataset.

Reply:
@magodo I've updated the docs for JSON dataset